srcpkgs/ulogd/template

```diff
@@ -13,10 +13,14 @@ license="GPL-2.0-only"
 homepage="http://www.netfilter.org/projects/ulogd/"
 distfiles="${homepage}/files/${pkgname}-${version}.tar.bz2"
 checksum=990a05494d9c16029ba0a83f3b7294fc05c756546b8d60d1c1572dc25249a92b
-CFLAGS="-D_GNU_SOURCE"
+conf_files="/etc/ulogd.conf /etc/logrotate.d/ulogd"
+make_dirs="/var/log/ulogd 0750 root adm"
```

Comment:
Can we make the directory setgid (2750 instead of 0750)?

>if a directory’s set-group-ID bit is set, newly created subfiles inherit the same group as the directory, and newly created subdirectories inherit the set-group-ID bit of the parent directory

https://www.gnu.org/software/coreutils/manual/html_node/Directory-Setuid-and-Setgid.html